themes/engineering/lib/scss/page/issues/_issues_list.scss

```diff
@@ -0,0 +1,26 @@
+@charset "UTF-8";
+
+.view.magazine-issues {
```

Comment:
Oh, and can you break up the two classes? Or maybe only use one? 
```.view { 
&.magazine-issues {
}}```

Reply:
So that's a workaround, since issues have different numbers of stories; the groups don't break evenly. this needs a slightly more in-depth refactor. Also the fact there's no way to browse through old issues arbitrarily.